index.html

```diff
@@ -19,25 +19,46 @@
     <!-- Enables media queries on mobile -->
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
+    <!-- Font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap" rel="stylesheet">
+    <style>@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;300;400;500;700;900&family=Zen+Dots&display=swap');</style>
+
     <!-- CSS-->
     <link type="text/css" rel="stylesheet" href="css/main.css" />
+    <link type="text/css" rel="stylesheet" href="css/welcome.css" />
 
     <title>Regis Altare Booster Pack Simulator</title>
   </head>
 
   <body>
     <main>
-      <h1>The Regis Altare Birthday Project 2023</h1>
-
-      <nav>
-        <a class="active" href="pages/collection.html">Collection</a>
-        <a href="pages/gacha.html">Gacha</a>
-        <a href="pages/message-board.html">Birthday Message Board</a>
-      </nav>
+      <div id="loading-overlay" class="active">
+        <span aria-live="polite">Loading</span>
+        <span aria-hidden="true">.</span>
+        <span aria-hidden="true">.</span>
+        <span aria-hidden="true">.</span>
+      </div>
 
-      <section id="card-list">
+      <section id="welcome-message" class="welcome-splash" aria-live="polite">
+        <h2>Happy Birthday Altare!</h2>
+        <h3>Let's see what the Altventurers have made for you</h3>
+        <!-- TODO: change this to an svg or img of an arrow -->
+        <button id="welcome-splash-button" type="button" onclick="toggleWelcomeTwo()">Enter site</button>
       </section>
 
+      <section id="welcome-fancy-menu" class="welcome-splash" aria-live="polite">
+        <img id="altare-img" src="images/AltareSplash.png" alt="A full screen splash of Altare jumping down from above"/>
+        <span class="visually-hidden">Select the following navigation options:</span>
```

Comment:
(nit) The `nav` tag is self explanatory (I hear VO say something like "start of navigation" and "end of navigation") so I don't think this visually hidden text is needed
Oh unless the idea was for this to auto-narrate when the menu loads in? That would make sense

Reply:
Yes, that was my intention to auto-narrate when the menu loads in.